h2o-py/h2o/model/model_base.py

```diff
@@ -755,6 +755,226 @@ def _plot(self, timestep, metric, **kwargs):
+            ax.spines['right'].set_visible(False)
+            ax.spines['top'].set_visible(False)
+            ax.spines['bottom'].set_color('#7B7B7B')
+            ax.spines['left'].set_color('#7B7B7B')
+            # Only show ticks on the left and bottom spines
+            ax.yaxis.set_ticks_position('left')
+            ax.xaxis.set_ticks_position('bottom')
+            plt.yticks(pos[0:num_of_features], feature_labels[0:num_of_features])
+            ax.margins(y=0.5)
+
+        else:
+            plt.barh(pos[0:num_of_features], val[0:num_of_features], align='center',
+                 height=0.8, color='#1F77B4', edgecolor='none')
+            # Hide the right and top spines, color others grey
+            ax.spines['right'].set_visible(False)
+            ax.spines['top'].set_visible(False)
+            ax.spines['bottom'].set_color('#7B7B7B')
+            ax.spines['left'].set_color('#7B7B7B')
+            # Only show ticks on the left and bottom spines
+            ax.yaxis.set_ticks_position('left')
+            ax.xaxis.set_ticks_position('bottom')
+            plt.yticks(pos[0:num_of_features], feature_labels[0:num_of_features])
+            ax.margins(y=0.5)
+
+        # check which algorithm was used to select right plot title
+        if self._model_json["algo"] == 'gbm':
+            plt.title("Variable Importance: H2O GBM", fontsize=20)
+            if not ('server' in list(kwargs.keys()) and kwargs['server']): plt.show()
+        elif self._model_json["algo"] == 'drf':
+            plt.title("Variable Importance: H2O DRF", fontsize=20)
+            if not ('server' in list(kwargs.keys()) and kwargs['server']): plt.show()
+        # if H2ODeepLearningEstimator has variable_importances = True
+        elif self._model_json["algo"] == 'deeplearning':
+            plt.title("Variable Importance: H2O Deep Learning", fontsize=20)
+            if not ('server' in list(kwargs.keys()) and kwargs['server']): plt.show()
+        else:
+            raise ValueError("A variable importances plot is not implemented for this type of model")
+
+
+    def std_coef_plot(self, num_of_features=None, **kwargs):
+        """
+        plot a GLM model's standardized coefficient magnitudes.
+
+        :param num_of_features: the number of features shown in the plot.
+
+        :returns: None.
+        """
+        # check for matplotlib. exit if absent.
+        try:
+            imp.find_module('matplotlib')
+            import matplotlib
+            if 'server' in list(kwargs.keys()) and kwargs['server']: matplotlib.use('Agg', warn=False)
+            import matplotlib.pyplot as plt
+        except ImportError:
+            print("matplotlib is required for this function!")
+            return
+
+        # check for six. exit if absent
+        try:
+            imp.find_module('six')
+            import six
+            if 'server' in list(kwargs.keys()) and kwargs['server']: six.use('Agg', warn=False)
+            from six import iteritems
```

Comment:
we don't require `six` anywhere. Instead of iteritems you can use the `viewitems` function